CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,417 @@
+# v1.4.0 - Changelog since v1.3.4
+
+## Changes by Kind
+
+### Feature
+
+- Updates Kubernetes dependencies to v1.22.0 (#814, @chrishenzie)
```

Comment:
[#814](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/878), same for the contributor names

Reply:
oh, I see. I thought it would be automatically rendered. Will add all the links.

Reply:
Just added all the links.